frontend/src/components/Tournament.js

```diff
@@ -44,6 +47,25 @@ const Tournament = () => {
     setTimeout(() => initFlowbite(), 8000);
   });
 
+  createEffect(() => {
```

Comment:
So I can see a few bugs in this.. So here player teams should not be considered as the source of truth as that contains all the teams a player has played for... So just for me I have L2F, BPHC and Doesnt Matter in my teams. So if BPHC and L2F comes like for last regionals then currently hub might show me as my team - BPHC instead of hub

Reply:
Also thing to note, for example ATC and CFT Alex will be the team admin for both but play only for CFT then you need to take that case as well

Reply:
> So I can see a few bugs in this.. So here player teams should not be considered as the source of truth as that contains all the teams a player has played for... So just for me I have L2F, BPHC and Doesnt Matter in my teams. So if BPHC and L2F comes like for last regionals then currently hub might show me as my team - BPHC instead of L2F

Interesting, i didn't consider the case when 2 of the player's teams would be in the tournament

Reply:
> Also thing to note, for example ATC and CFT Alex will be the team admin for both but play only for CFT then you need to take that case as well

a person can only play on 1 team right ? So do you suggest we show the playing team and non-playing teams where the person can submit scores ? So in this case, we'll show "Playing team: CFT" and "Non-playing team: ATC". could have name other than "non-playing team" though

Reply:
> Hmm I feel we can just show playing team... Non playing team is not required during tournament. It is just for like creating roster and all before tournament

there could be non-playing admin/coach can submit scores right ? also non-playing assistant coach is there

Reply:
Oh yeah true thats there.. then maybe return list of team_ids just like the current /me/access has and show the whole list.. like make a button card for every team in team_ids and show like "your team" same stull only.. dont need  to differentiate playing / non playing then